azurerm/resource_arm_virtual_machine_unmanaged_disks_test.go

```diff
@@ -81,6 +81,25 @@ func TestAccAzureRMVirtualMachine_basicLinuxMachineSSHOnly(t *testing.T) {
 	})
 }
 
+func TestAccAzureRMVirtualMachine_basicLinuxMachine_withWriteAcceleratorEnabled(t *testing.T) {
```

Comment:
so this test currently fails:

```
        * azurerm_virtual_machine.test: compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=409 -- Original Error: failed request: autorest/azure: Service returned an error. Status=<nil> Code="SkuNotAvailable" Message="The requested size for resource '/subscriptions/*******/resourceGroups/acctestRG-6429603035887554193/providers/Microsoft.Compute/virtualMachines/acctvm-6429603035887554193' is currently not available in location 'westeurope' zones '' for subscription '*******'. Please try another size or deploy to a different location or zones. See https://aka.ms/azureskunotavailable for details."
```

Judging by the error message - I'm wondering if a Zone needs to be set?

Reply:
It looks like it is only availiable in the following regions:
- West US2
- East US2
- Western Europe
- Southeast Asia